x/blob/client/cli/payforblob.go

```diff
@@ -34,8 +34,11 @@ func CmdPayForBlob() *cobra.Command {
 		Use:   "PayForBlobs [hexNamespaceID] [hexBlob]",
 		Short: "Pay for a data blob to be published to the Celestia blockchain",
 		Long: "Pay for a data blob to be published to the Celestia blockchain. " +
-			"[hexNamespaceID] must be a 10 byte hex encoded namespace ID. " +
+			"[hexNamespaceID] must be a 10 byte hex encoded namespace ID, " +
+			"(for the namespace version 0). " +
+			"The hex string should not include a 0x prefix. " +
 			"[hexBlob] can be an arbitrary length hex encoded data blob. " +
+			"The hex string should not include a 0x prefix. " +
 			"This command only supports a single blob per invocation. ",
```

Comment:
1. [not blocking] We can add support for parsing the `0x` prefix if one is provided. This could be a follow up issue.
2. [optional] Instead of repeating "The hex string should not include a 0x prefix." twice, this could state it once like:

```suggestion
			"[hexNamespaceID] must be a 10 byte hex encoded namespace ID for the namespace version 0. " +
			"[hexBlob] can be an arbitrary length hex encoded data blob. " +
			"Hex strings should not include a 0x prefix. " +
			"This command only supports a single blob per invocation. ",
```

Reply:
> [not blocking] We can add support for parsing the 0x prefix if one is provided. This could be a follow up issue.

Yes, that would be an option too. 

> [optional] Instead of repeating "The hex string should not include a 0x prefix." twice, this could state it once like:

Applied your suggestion https://github.com/celestiaorg/celestia-app/pull/2389/commits/6654226db2f024eb3a432e2c199f016c774a88a6

Reply:
> 1. [not blocking] We can add support for parsing the `0x` prefix if one is provided. This could be a follow up issue.

Please see that tracking issue https://github.com/celestiaorg/celestia-app/issues/2392